internal/plugins/helm/v1/chartutil/chart.go

```diff
@@ -114,22 +117,23 @@ type CreateOptions struct {
 //
 // CreateChart returns an error if an error occurs creating the scaffold.Resource or
 // creating the chart.
-func CreateChart(projectDir string, opts CreateOptions) (*resource.Options, *chart.Chart, error) {
+func CreateChart(projectDir string, opts CreateOptions) (*golang.Options, *chart.Chart, error) {
 	chartsDir := filepath.Join(projectDir, HelmChartsDir)
 	err := os.MkdirAll(chartsDir, 0755)
 	if err != nil {
 		return nil, nil, fmt.Errorf("failed to create helm-charts directory: %v", err)
 	}
 
 	var (
-		r *resource.Options
+		r *golang.Options
 		c *chart.Chart
 	)
 
+	r = &golang.Options{}
 	// If we don't have a helm chart reference, scaffold the default chart
 	// from Helm's default template. Otherwise, fetch it.
 	if len(opts.Chart) == 0 {
-		r, c, err = scaffoldChart(chartsDir, opts.GVK.Group, opts.GVK.Version, opts.GVK.Kind)
+		r, c, err = scaffoldChart(chartsDir, opts.Domain, opts.GVK.Kind, opts.GVK.Version, opts.GVK.Kind)
```

Comment:
Do you really mean to pass the Kind twice? Or is one of them supposed to be Group?
```suggestion
		r, c, err = scaffoldChart(chartsDir, opts.Domain, opts.GVK.Group, opts.GVK.Version, opts.GVK.Kind)
```